src/unraphael/dash/align.py

```diff
@@ -466,3 +469,158 @@ def align_image_to_base(
         raise ValueError(f'No such method: {align_method}')
 
     return func(image=image, base_image=base_image, **kwargs)
+
+
+def feature_based_alignment_visual(
+    base_image: np.ndarray,
+    target_image: np.ndarray,
+    method: str,
+    base_image_name: str,
+    target_image_name: str,
+    display_in_grayscale: bool = True,
+    max_ratio: float = 0.6,
+):
+    """Visualize feature-based alignment between two images using a specified
+    feature detection method.
+
+    Parameters
+    ----------
+    base_image : np.ndarray
+        The base image to be used as a reference for alignment
+    target_image : np.ndarray
+        The target image to be aligned with the base image
+    method : str
+        The feature detection method to use for alignment ('SIFT' or 'ORB')
+    base_image_name : str
+        Name of the base image for labeling
+    target_image_name : str
+        Name of the target image for labeling
+    display_in_grayscale : bool, optional
+        Whether to display images in grayscale or original color, by default True
+    max_ratio : float, optional
+        The maximum ratio for descriptor matching, by default 0.6
+
+    Returns
+    -------
+    None
+        Displays a plot showing the feature-based alignment
```

Comment:
This is not necessary, but please update the function name/documentation that this will plot the aligment

```suggestion
```